guides/v2.3/graphql/mutations/create-customer-address.md

```diff
@@ -95,6 +95,17 @@ The `createCustomerAddress` mutation returns the following attributes:
 
 {% include graphql/customer-address-output.md %}
 
+## Errors
+
+Error | Description
+--- | ---
+`Expected type CustomerAddressInput!, found "".` | The `input` attribute contains an empty value.
+`"input" value should be specified` | The `input` attribute is set but contains an empty data.
```

Comment:
Either change the words after "contains" to "empty data" or "an empty value".

Reply:
`Expected type CustomerAddressInput!, found "".` ties to this mutation:

```
mutation {
  createCustomerAddress(
    input: ""
  ) {
    city
  }
}
```

`"input" value should be specified` appears in:
```
mutation {
  createCustomerAddress(
    input: {}
  ) {
    city
  }
}
```